### PR TITLE
[CPS-708] feat(*): Support install containerd server at alone host

### PR DESCRIPTION
**Pull request by WangMinjie114**
**What type of PR is this?**
/kind feature
/area release

**What this PR does / why we need it**:
因 Cargo 与 ES 部署需求，需要支持在独立服务器节点安装 containerd 服务。

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @bbbmj @liwandaniel @hyy0322 @jianglinjian 

**Does this PR introduce a user-facing change?**:

```release-note
Config yum mirrors and install containerd service at alone host.
```

### CODE REVIEW

**Comment by hyy0322** [comment]
LGTM

**Comment by jianglinjian** [comment]
LGTM

File: hack/infra/sources/infra_init.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+## FileName: install_containerd.sh
```
Comment (supereagle) [inline hack/infra/sources/infra_init.sh]: The filename of this script is different from it in this comment.

File: hack/infra/sources/infra_init.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    fi
+else
+    echo -e "$RED_COL The CPU architecture is not x86 $NORMAL_COL"
+fi
+if [[ ${RELEASE} == '' ]]; then
+    echo -e "$RED_COL Can not get system message, please check $NORMAL_COL"
+    exit 1
+fi
+
+# Stop firewalld
```
Comment (bbbmj) [inline hack/infra/sources/infra_init.sh:43]: 下面的公共代码抽出来

**Review by bbbmj — COMMENTED** [review]
General LGTM
/approve

File: hack/infra/sources/infra_init.sh (hunk 2 above)
Comment (WangMinjie114) [reply hack/infra/sources/infra_init.sh:43]: 与 @bbbmj 讨论后，由于拆出来后无法复用脚本。因此还是保留现有的状态。

File: hack/infra/sources/infra_init.sh (hunk 1 above)
Comment (WangMinjie114) [reply hack/infra/sources/infra_init.sh]: fixed

**Review by bbbmj — COMMENTED** [review]
/lgtm

**Pull request merged**